src/client.h

```diff
@@ -427,6 +436,8 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	bool sendModChannelMessage(const std::string &channel, const std::string &message);
 	ModChannel *getModChannel(const std::string &channel);
 
+	const ServerInfo *getServerInfo() const { return &m_server_info; }
```

Comment:
generally we use const ref not const pointer

Reply:
so just return a reference instead?

Reply:
yeah just a const ref :)

Reply:
Resolved?